src/main/kotlin/no/nav/syfo/service/Oppgaveservice.kt

```diff
@@ -45,7 +46,7 @@ suspend fun handleRegisterOppgaveRequest(
         } catch (ex: ServerResponseException) {
```

Comment:
Kan vi i begge tilfellene få printet til loggen hva ex var for noe (ex.message eller noe slikt)? Helst sammen med loggingmeta slik at vi ser hvilken melding som trigget feilen? :)